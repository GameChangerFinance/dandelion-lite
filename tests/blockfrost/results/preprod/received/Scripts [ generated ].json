[
  {
    "script_hash": "c0f20d4d718e5991dccc855c6c80cee620e7504a568211bae32f2dd3"
  },
  {
    "script_hash": "924bbdd907aced43f10923f83e93051f1cccb9538a8a2b84e16b8501"
  },
  {
    "script_hash": "6cf09305d56f24262e0bbda829d7583c70827838400eef30aeb1038f"
  }
]
