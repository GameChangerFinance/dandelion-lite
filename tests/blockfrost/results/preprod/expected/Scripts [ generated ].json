[
  {
    "script_hash": "3da3bdb8c7703cfcc6a4627fe080f57a7b189062f9d4e2f118d604ef"
  },
  {
    "script_hash": "76f7809377bf26bccdd5ab8670cf4e98a64b1acfaca15fb7577bcd08"
  },
  {
    "script_hash": "36c9ae65bc49f59d18a8ca58dc7660ab8d974ac00878ae8ee2503b97"
  }
]
